applications/ShapeOptimizationApplication/test_examples/05_External_Fluid_Adjoint_Transient_Analyzer/base_case/impose_parabolic_inlet_process.py

```diff
@@ -0,0 +1,31 @@
+import KratosMultiphysics
```

Comment:
I think the InletProcess in Fluid is able to do what you do with this process
@jcotela @rubenzorrilla ?